tests/sentry/api/validators/sentry_apps/util.py

```diff
@@ -16,7 +16,6 @@ def decorator(func):
         def inner(self, *args, **kwargs):
             with self.assertRaises(ValidationError) as cm:
                 func(self)
-            # assert cm.exception.message == message
             found_message = cm.exception.message
```

Comment:
Probably need `text_type` here?

Reply:
No, `cm.exception` is a `jsonschema.ValidationError` which has a message attrib.

https://github.com/getsentry/sentry/pull/16731/commits/94718748329c8329d11a480cbb4274e1076127a2